Bottom: b02719446e886b28c18d2efb7bd634f34530e8d8
Top:    130e728397db395384dc91ce21747c590dedb3c2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-10 17:06:42 +0000

Refresh of revert-do-not-warp-pointer-too

---

diff --git a/src/hid.h b/src/hid.h
index aeadef0..b08c65f 100644
--- a/src/hid.h
+++ b/src/hid.h
@@ -527,7 +527,7 @@ typedef enum
 
 #define HID_SC_PAN_VIEWPORT                        2
 
-#define HID_SC_CENTER_IN_VIEWPORT                  3
+#define HID_SC_CENTER_IN_VIEWPORT_AND_WARP_POINTER  3
 
       hidval (*add_timer) (void (*func) (hidval user_data_),
 			   unsigned long milliseconds_, hidval user_data_);
